ivy/functional/ivy/random.py

```diff
@@ -46,7 +46,7 @@ def _check_bounds_and_get_shape(low, high, shape):
         return ivy.shape(low)
     if isinstance(high, valid_types):
         return ivy.shape(high)
-    return ivy.Shape(())
+    return ()
```

Comment:
Also, in order to merge the PR you'll have to remove changes in files that are not related to the issue you are solving, i.e.  only `discrete_fourier_transform.py` and `test_discrete_fourier_transform.py` should be affected.